### PR TITLE
VORTEX-5826 Feature layer icon showing as ? in the layers window

### PR DESCRIPTION
**Pull request by Xenotide**
Fixes #
- CSVs with no SPATIAL columns as a result of no data are now given MapVisualization anyway

### CODE REVIEW

File: open-sphere-base/control-panels/src/main/java/io/opensphere/controlpanels/layers/util/FeatureTypeLabel.java [hunk 1]
```diff
@@ -338,6 +338,12 @@ public void setIconByType(DataTypeInfo dti)
             MapVisualizationType type = dti.getMapVisualizationInfo().getVisualizationType();
             setIconByType(dti.getBasicVisualizationInfo().getTypeColor(), type);
         }
+        else if (dti != null && dti.getClass().getSimpleName().equals("CSVDataTypeInfo"))
```
Comment (brelsftm) [inline open-sphere-base/control-panels/src/main/java/io/opensphere/controlpanels/layers/util/FeatureTypeLabel.java]: do an `instanceof` CSVDataTypeInfo

Comment (roddyaj) [reply open-sphere-base/control-panels/src/main/java/io/opensphere/controlpanels/layers/util/FeatureTypeLabel.java]: I don't think Control Panels should know anything about CSV

File: open-sphere-plugins/csv/src/main/java/io/opensphere/csv/CSVTypeInfoGenerator.java [hunk 2]
```diff
@@ -77,7 +78,8 @@ public static CSVDataTypeInfo generateTypeInfo(Toolbox tb, CSVDataSource fileSou
         typeInfo.applyColorPreferences();
 
         // Set map visualization info
-        if (fileSource.getParseParameters().hasCategory(ColumnType.Category.SPATIAL))
+        if (fileSource.getParseParameters().hasCategory(ColumnType.Category.SPATIAL)
+                || fileSource.getParseParameters() instanceof CSVParseParameters)
```
Comment (roddyaj) [inline open-sphere-plugins/csv/src/main/java/io/opensphere/csv/CSVTypeInfoGenerator.java]: What's the point of checking instanceof CSVParseParameters?  I thought that would always be true.

Comment (Xenotide) [reply open-sphere-plugins/csv/src/main/java/io/opensphere/csv/CSVTypeInfoGenerator.java]: The idea was to let CSVs that don't have a SPATIAL column loaded still have MapVisualization loaded. The would let, for example, a CSV with only column headings and no data be loaded in as a feature and not an unknown. I wasn't sure if the instanceof check would always be true. I could look into seeing if there are other parameters that could fit instead, or just do away with the conditional in the extreme case if that wouldn't break something.

**Pull request merged**